### PR TITLE
Remove dot from release when needed

### PR DESCRIPTION
**Pull request by fgimenez**
Prerequisite: #26
We only need to remove the dot from the release for cloud related queries, system-image needs it.

### CODE REVIEW

File: pkg/runner/runner.go [hunk 1]
```diff
@@ -86,7 +88,8 @@ func (r *Runner) create(options *flags.Options) (err error) {
 		return &ErrVersion{siVersion, cloudVersion}
 	}
 	var path string
-	path, err = r.imgDriver.Create(options.Release, options.Channel, options.Arch, siVersion)
+	dotRelease := addDots(options.Release)
```
Comment (come-maiz) [inline pkg/runner/runner.go]: I would add a comment here saying that the dot is only added if the release is a number of 4 digits, like 15.04

File: pkg/runner/runner.go [hunk 2]
```diff
@@ -146,3 +152,12 @@ func (r *Runner) cleanup(options *flags.Options) (err error) {
 	}
 	return
 }
+
+func addDots(release string) string {
```
Comment (come-maiz) [inline pkg/runner/runner.go]: There's only one dot, so s/addDots/addDot.

**Comment by come-maiz** [comment]
Just a minor change to make related to there being only one dot.

**Comment by fgimenez** [comment]
Thanks @elopio, fixed

**Pull request merged**